pkg/testworkflows/testworkflowprocessor/processor.go

```diff
@@ -384,6 +386,12 @@ func (p *processor) Bundle(ctx context.Context, workflow *testworkflowsv1.TestWo
 	if podConfig.SecurityContext.FSGroup == nil {
 		podConfig.SecurityContext.FSGroup = common.Ptr(constants.DefaultFsGroup)
 	}
+	hostPID := false
+	if options.AllowLowSecurityFields {
+		if podConfig.HostPID != nil {
+			hostPID = *podConfig.HostPID
```

Comment:
Shouldn't it throw error instead if not allowed?

Reply:
agree, should be better